notebooks/niriss_ami_binary/2_niriss_ami_binary.ipynb

```diff
@@ -11,19 +11,21 @@
    "cell_type": "markdown",
```

Comment:
<p>The first two lines in this cell aren't needed since <code>datafiles</code> was already defined earlier.</p><p> </p><p>You can handle the lines in the loop that use <code>df.replace</code> like this: <code>outdir + df.name.replace('uncal', 'rateints')</code>.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Done, thanks!

Reply:
Sorry, looks like I had a typo here, too. If `outdir` is a `Path` object, the replacement should be `outdir / df.name.replace('uncal', 'rateints')` (slash instead of plus sign).